src/WTelegramClient.csproj

```diff
@@ -28,12 +28,24 @@
   <ItemGroup>
     <None Remove=".gitattributes" />
     <None Remove=".gitignore" />
-    <!--<None Include="..\README.md" Pack="true" PackagePath=""/>-->
+    <None Remove="azure-pipelines.yml" />
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="ArrayDeconstructors.Source" Version="1.0.3">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
     <PackageReference Include="Crc32.NET" Version="1.2.0" />
+    <PackageReference Include="IndexRange" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Bcl.HashCode" Version="1.1.1" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
I forgot to delete this, just delete Newtonsoft.Json package and forget about this :)